doc/nrf/releases_and_maturity/releases/release-notes-2.1.0.rst

```diff
@@ -269,7 +269,7 @@ nRF9160: Asset Tracker v2
   * LwM2M schema.
     To use the new parameters, upload the updated :file:`config_object_descript.xml` file to AVSystem.
   * The conversions of RSRP and RSRQ.
-    These now use common macros that follow the conversion algorithms defined in the `AT Commands Reference Guide`_.
+    These now use common macros that follow the conversion algorithms defined in the AT Commands Reference Guide.
```

Comment:
Or should we refer to 9160 document in these old references? You can decide with your team mates.

Reply:
Discussed with team and kept both the old and new name to this link. So, that old release notes need not be touched and added an entry in the changelog about the change.

Reply:
This looks good.